index.html

```diff
@@ -212,38 +212,9 @@ <h2>
         actually an interface, is sometimes used instead of the more accurate
         "an object implementing the interface <code>Foo</code>.
       </p>
-      <p>
-        The term <dfn>DOM</dfn> is used to refer to the API set made available
-        to scripts in Web applications, and does not necessarily imply the
-        existence of an actual [=Document=] object or of any other [=Node=]
-        objects as defined in the [[DOM]] specification.
-      </p>
-      <p>
-        A DOM attribute is said to be <dfn>getting</dfn> when its value is
-        being retrieved (such as by author script), and is said to be
-        <dfn>setting</dfn> when a new value is assigned to it.
-      </p>
-      <p>
-        The term <dfn>JavaScript</dfn> is used to refer to ECMA262, rather than
-        the official term ECMAScript, since the term JavaScript is more widely
-        known. [[ECMASCRIPT]]
-      </p>
-      <p>
-        The term <dfn>resource</dfn> is used to refer to elements and any other
```

Comment:
It seems fine to delete this but does this API still have a clear explanation about what resources are exposed via ResourceTiming?

Reply:
https://w3c.github.io/resource-timing/#resources-included-in-the-performanceresourcetiming-interface seems to cover it